terminal_check_unix.go

```diff
@@ -1,4 +1,5 @@
 // +build linux aix
+// +build !appengine
```

Comment:
I think this single change is enough to fix your build issue, can you try ?

Reply:
This would fix the build problem on *my* machine, because I'm running Linux.

However, if one wants to build and deploy AppEngine projects from various BSD or Solaris, they'd need the other two `// +build !appengine` constraints in the corresponding files.

Reply:
I guess the tag when building for an appengine are always linux and appengine ?
The host build doesn't matter really here, we just want to differentiate standard linux build for linux+appengine target.
Or am I wrong somewhere ?